hypervisor/src/cpu.rs

```diff
@@ -244,6 +246,8 @@ pub enum VmExit<'a> {
     Reset,
     Shutdown,
     Hyperv,
+    #[cfg(feature = "kvm")]
+    Debug(kvm_bindings::kvm_debug_exit_arch),
```

Comment:
Is this the right approach here? You're exposing KVM details to common code.

At the very least, can you please reexport this struct using a common name?

Reply:
I initially intended to expose this info for future development, but it is currently not used. I removed it with 99ec2c8.